ballerina/commons.bal

```diff
@@ -53,25 +94,26 @@ public enum WaitingPolicy {
   LOG_AND_IGNORE
 }
 
-# Gets time in milliseconds of the given `time:Civil`.
-#
-# + time - The Ballerina `time:Civil`
-# + return - Time in milliseconds or else `task:Error` if the process failed due to any reason
-public isolated function getTimeInMillies(time:Civil time) returns int|Error {
-    if time[UTC_OFF_SET] == () {
-        time:ZoneOffset zoneOffset = {hours: 0, minutes: 0};
-        time.utcOffset = zoneOffset;
-    }
-    time:Utc|time:Error utc = time:utcFromCivil(time);
-    if utc is time:Utc {
-        if utc > time:utcNow() {
-            return <int> decimal:round((<decimal>utc[0] + utc[1]) * 1000);
-        }
-        return error Error(
-                    string `Invalid time: ${time.toString()}. Scheduled time should be greater than the current time`);
-    } else {
-        return error Error(string `Couldn't convert given time to milli seconds: ${utc.message()}.`);
-    }
-}
+class TimeConverter {
```

Comment:
I assume this is only used to convert a `time:Civil` into a milliseconds from the native-code. Have we explored the possibility to use ballerina time-native APIs to do that rather than writing a logic here ?

Reply:
There was a function to directly call a function from native side.  Used that approach here. 

`env.getRuntime().callFunction(...//)`